aries-site/src/examples/components/tooltip/ToolTipExitExample.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, Button, Tip, Text } from 'grommet';
+import { FormClose } from 'grommet-icons';
+
+export const ToolTipExitExample = () => {
+  return (
+    <Box align="start" justify="center" fill>
+      <Box gap="small" direction="row">
+        <Box width="small">
+          <Text color="text-strong" size="large" weight="bold">
+            Add Service
+          </Text>
+          <Text>
+            Visit the HPE catalog to start addiing all of your favoriite
+            services.
+          </Text>
+        </Box>
+        <Box>
```

Comment:
Is this Box necessary here? Can we get rid of it or replace is with a React fragment?